_episodes/01-raster-structure.md

```diff
@@ -15,10 +15,474 @@ objectives:
+        309.25      , 312.01998901],
+       [369.95999146, 372.6000061 , 372.42999268, ..., 316.38998413,
+        309.86999512, 311.20999146]])
+~~~
+{: .output}
+
+The output of `.read()` is a numpy array, which is abbreviated with `...` since the array is too large for the output console.
+
+To visualise this data in Python using `earthpy.plot`, all we need is our data in a numpy array and some options to control
+the color of our plot and plot labels.
+
+~~~
+%matplotlib inline
+earthpy.plot.plot_bands(
+    surface_model_HARV_arr,
+    scale=False,
+    cmap="viridis",
+    title="Digital Surface Model Without Hillshade",
+    figsize=(10, 6)
+)
+~~~
+
+
+Nice plot! We set the color scale to `viridis` which is a color-blindness friendly color scale.
+
+<img src="../fig/01-earthpy-surface-1.png" title="Raster plot with earthpy.plot using the viridis color scale" alt="Raster plot with earthpy.plot using the viridis color scale" width="612" style="display: block; margin: auto;" />
+
+> ## Plotting Tip
+> For more aesthetic looking plots, matplotlib allows you to customize the style with `plt.style.use`. However, if you want more control of the look of your plot, matplotlib has many more functions to change the postion and appearnce of plot elements.
+> > ## Show plot
+> >  Here is the result of using a ggplot like style for our surface model plot.
+> > 
+> > ~~~
+> > import matplotlib.pyplot as plt
+> > %matplotlib inline
+> > plt.style.use("ggplot")
+> > earthpy.plot.plot_bands(
+> >     surface_model_HARV_arr,
+> >     scale=False,
+> >     cmap="viridis",
+> >     title="Digital Surface Model Without Hillshade",
+> >     figsize=(10, 6)
+> > )
+> > ~~~
+> > {: .language-python}
+> > 
+> > <img src="../fig/01-earthpy-ggplot-style-2.png" title="plot of chunk unnamed-chunk-5" alt="plot of chunk unnamed-chunk-5" width="612" style="display: block; margin: auto;" />
+> {: .solution}
+{: .callout}
+
+This map shows the elevation of our study site in Harvard Forest. From the
+legend, we can see that the maximum elevation is ~400, but we can't tell whether
+this is 400 feet or 400 meters because the legend doesn't show us the units. We
+can look at the metadata of our object to see what the units are. Much of the
+metadata that we're interested in is part of the CRS, and it can also be accessed 
+by calling other attributes besides `meta`. We introduced the concept of a CRS in [an earlier
+lesson](https://datacarpentry.org/organization-geospatial/03-crs) (TODO replace link).
+
+Now we will see how features of the CRS appear in our data file and what
+meanings they have.
+
+### View Raster Coordinate Reference System (CRS) in Python
+We can view the CRS string associated with our Python object using the`crs`
+attribute.
+
+
+~~~
+surface_model.crs
+~~~
+{: .language-python}
+
+
+
+~~~
+CRS.from_epsg(32618)
+~~~
+{: .output}
+
+You can convert the EPSG code to a PROJ4 string with `earthpy.epsg`, another python `dict` which maps epsg codes (keys) to
```

Comment:
This is somewhat convoluted/confusing, as you need to manually type the code from the crs created by rasterio to the string expected by earthpy.epsg. 

Should we instead be building up to:
```
earthpy.epsg[str(surface_model_HARV.crs.to_epsg())]
```
?

Perhaps by showing that you need a string input for earthpy.epsg, and then showing how to get the string from rasterio?